angular.json

```diff
@@ -142,64 +190,117 @@
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
           "options": {
-            "browserTarget": "packages:build"
+            "browserTarget": "visual-regression:build"
           },
           "configurations": {
             "production": {
-              "browserTarget": "packages:build:production"
+              "browserTarget": "visual-regression:build:production"
             }
           }
         },
         "extract-i18n": {
           "builder": "@angular-devkit/build-angular:extract-i18n",
           "options": {
-            "browserTarget": "packages:build"
+            "browserTarget": "visual-regression:build"
+          }
+        },
+        "test": {
+          "builder": "@angular-builders/jest:run",
+          "options": {
+            "main": "projects/visual-regression/src/test.ts",
+            "polyfills": "projects/visual-regression/src/polyfills.ts",
+            "tsConfig": "projects/visual-regression/tsconfig.spec.json",
+            "assets": [
+              "projects/visual-regression/src/favicon.ico",
+              "projects/visual-regression/src/assets"
+            ],
+            "styles": [
+              "projects/visual-regression/src/styles.scss"
+            ],
+            "scripts": []
           }
         },
         "lint": {
           "builder": "@angular-devkit/build-angular:tslint",
           "options": {
             "tsConfig": [
-              "demo/tsconfig.app.json",
-              "demo/tsconfig.spec.json"
+              "projects/visual-regression/tsconfig.app.json",
+              "projects/visual-regression/tsconfig.spec.json",
+              "projects/visual-regression/e2e/tsconfig.json"
             ],
-            "exclude": []
+            "exclude": [
+              "**/node_modules/**"
+            ]
+          }
+        },
+        "e2e": {
+          "builder": "@angular-devkit/build-angular:protractor",
+          "options": {
+            "protractorConfig": "projects/visual-regression/e2e/protractor.conf.js",
+            "devServerTarget": "visual-regression:serve"
+          },
+          "configurations": {
+            "production": {
+              "devServerTarget": "visual-regression:serve:production"
+            }
           }
         }
       }
     },
-    "packages-e2e": {
-      "root": "",
-      "sourceRoot": "",
-      "projectType": "application",
+    "library": {
```

Comment:
@atlwendy  this is still not the correct config for a CLI library. We should generate a library with the CLI and then move our files into that library project so that we can be sure everything is wired up correctly.